vscode/src/KonveyorGUIWebviewViewProvider.ts

```diff
@@ -213,7 +213,7 @@ export class KonveyorGUIWebviewViewProvider implements WebviewViewProvider {
 
     webview.onDidReceiveMessage(
       (message) => {
-        if (message.command === "webviewReady") {
+        if (message.command === "WEBVIEW_READY") {
```

Comment:
1. `message .type` ? 
2. webview seemed to work without this notification - do we still need it?

Reply:
The webview_ready is used by the panels - need to look at this when fixing the fullscreen analysis page.